docs/functions/functions.json

```diff
@@ -516,6 +588,23 @@
 				true
 			]
 		},
+		{
+			"name": "unique",
+			"id": "unique",
+			"unary": false,
+			"category": "lists",
+			"role": "function",
+			"description": "Creates a list with only unique items. Returns your parameter back if it’s not a list.",
```

Comment:
This is a bit confusing. I'd rephrase as "Drops any duplicates, returning only the unique items from a list of values. For example, `unique(list(1, 2, 1, 4, 4, 3, 2))` will return `list(1, 2, 3, 4)`.